app/celery/tasks.py

```diff
@@ -472,6 +477,9 @@ def update_letter_notifications_statuses(self, filename):
                 current_app.logger.info(
                     'DVLA file: {filename}, notification updated to {status}: {reference}'.format(
                         filename=filename, status=status, reference=str(update.reference)))
+                notifications = dao_get_notifications_by_reference(references=[update.reference])
+                for notification in notifications:
+                    send_delivery_status_to_service.apply_async([notification.id], queue=QueueNames.NOTIFY)
```

Comment:
Need to wrap notification_id with str(notification_id) otherwise there will be a serialisation error

Reply:
changed in 3594fc.